src/ompl/util/RandomNumbers.h

```diff
@@ -139,6 +146,30 @@ namespace ompl
             return localSeed_;
         }
 
+        /** \brief Uniform random sampling of a unit-length vector. I.e., the surface of an n-ball */
+        void uniformNormalVector(unsigned int n, double value[]);
```

Comment:
I've clarified the comment.

I had used value[] to be consistent with the other functions that return arrays (i.e., `quaternion(double value[4])`). Granted, they are all fixed size arrays, so maybe that justifies a stylistic difference? I don't know, please advise.